src/lib/i18n/de.ts

```diff
@@ -0,0 +1,161 @@
+    'note.createkeymessage3': 'um eine neue Notiz zu erstellen',
+    'note.restore': 'Wiederherstellen',
+
+    //Bookmark
+    'bookmark.remove': 'Lesezeichen entfernen',
+    'bookmark.add': 'Lesezeichen hinzufügen',
+
+    //About
+    'about.about': 'Über',
+    'about.boostnoteDescription':
+      'Ein quelloffenes Notiz-Programm geschaffen für Programmierer wie dir.',
+    'about.website': 'Offizielle Webseite',
+    'about.boostWiki': 'Boost Note für Team',
+    'about.platform': 'Cross-platform',
+    'about.community': 'Community',
+    'about.github': 'GitHub Repository',
+    'about.bounty': 'Bounty on IssueHunt',
+    'about.blog': 'Blog',
+    'about.slack': 'Slack Gruppe',
+    'about.twitter': 'Twitter Account',
+    'about.facebook': 'Facebook Gruppe',
+    'about.reddit': 'Reddit',
+
+    //Billing
+    'billing.billing': 'Billing',
+    'billing.message': 'Bitte melde dich an um deinen Plan zu ugpraden.',
+    'billing.basic': 'Basic',
+    'billing.current': 'Aktuell',
+    'billing.premium': 'Premium',
+    'billing.price': '$3/Monat (USD) *',
+    'billing.browser': 'Browser App',
+    'billing.desktop': 'Desktop App (Mac/Windows/Linux)',
+    'billing.mobile': 'Mobile App (Wird gelauncht am Jan, 2020)',
+    'billing.sync': 'Mehrere Geräte synchronisieren',
+    'billing.local': 'Lokaler Speicher',
+    'billing.cloud': 'Cloud Speicher',
+    'billing.storageSize': 'Cloud Speicher Größe',
+    'billing.addStorageDescription':
+      '* Wenn du mehr Cloud-Speicher brauchst, zahlst du 5$ (USD) für jede zusätzliche 5GB. Drücke auf den "Extra Speicher hinzufügen"-Knopf unten.',
+    'billing.addStorage': 'Extra Speicher hinzufügen',
+
+    // Preferences
+    'preferences.general': 'Einstellungen',
+
+    // Preferences GeneralTab
+    'preferences.account': 'Account',
+    'preferences.addAccount': 'Anmelden',
+    'preferences.loginWorking': 'Wird angemeldet...',
+    'preferences.interfaceLanguage': 'Oberflächen Sprache',
+    'preferences.applicationTheme': 'Anwendungs-Theme',
+    'preferences.auto': 'Auto',
+    'preferences.light': 'Hell',
+    'preferences.dark': 'Dunkel',
+    'preferences.sepia': 'Sepia',
+    'preferences.solarizedDark': 'Solarized Dark',
+    'preferences.noteSorting': 'Notiz-Sortierung',
+    'preferences.dateUpdated': 'Datum aktualisiert',
+    'preferences.dateCreated': 'Datum erstellt',
+    'preferences.title': 'Titel',
+    'preferences.analytics': 'Analytik',
+    'preferences.analyticsDescription1':
+      'Boostnote sammelt anonyme Daten um das Nutzererlebnis zu verbessern, und es wird streng darauf geachtet keine persönlichen Informationen wie die Inhalte der Notizen zu senden. Um zu sehen wie das läuft, besuche unsere Github Seite.',
+    'preferences.analyticsDescription2':
+      'Du kannst wählen diese Option zu aktivieren oder deaktivieren.',
+    'preferences.analyticsLabel': 'Analytik aktivieren um mitzuhelfen Boostnote zu verbessern',
```

Comment:
my suggestion: Analytik aktivieren um dich an der Verbesserung von Boostnote zu beteiligen.

i leave it to you